subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -449,6 +780,10 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_commands,
 	SHELL_CMD(scan, NULL, "Scan for Wi-Fi APs", cmd_wifi_scan),
 	SHELL_CMD(status, NULL, "Status of the Wi-Fi interface", cmd_wifi_status),
 	SHELL_CMD(statistics, NULL, "Wi-Fi interface statistics", cmd_wifi_stats),
+	SHELL_CMD(ps, NULL, "Configure Wi-Fi power save on/off, no arguments will dump config",
+		  cmd_wifi_ps),
+	SHELL_CMD(ps_mode, NULL, "Configure Wi-Fi power save mode legacy/wmm", cmd_wifi_ps_mode),
+	SHELL_CMD(twt, &wifi_twt_ops, "Manage TWT flows", NULL),
```

Comment:
Very minor nit, but could the commands be set in alphabetical order, it does not change the behaviour but is easier to find if/when we get more commands.